axelrod/tests/unit/test_player.py

```diff
@@ -43,7 +46,21 @@ def test_noisy_play(self):
         self.assertEqual(p1.history[0], 'D')
         self.assertEqual(p2.history[0], 'D')
 
-C, D = 'C', 'D'
+
+class MockPlayer(axelrod.Player):
+    """Creates a mock player that enforces a particular next move for a given
+    player."""
+
+    def __init__(self, player, move):
+        # Need to retain history for opponents that examine opponents history
+        # Do a deep copy just to be safe
+        self.history = copy.copy(player.history)
```

Comment:
The MockPlayer needs to have the right history in case the opponent inspects or manipulates it -- I made it a deep copy in case some of cheating strategies mess with the history.